README.md

```diff
@@ -179,3 +179,26 @@ Also included is:
 - **Driver utilities**, `chisel3.Driver`, which contains compilation and test
   functions that are invoked in the standard Verilog generation and simulation
   testing infrastructure. These can also be used as part of custom flows.
+
+### Which version should I use?
+
+The chisel eco-system (`chisel3`, `firttl`, `dsptools`, `firrtl-interpreter`, `treadle`, `diagrammer`) use semantic versioning:
```

Comment:
There are semantics to our versions, but we don't use "Semantic Versioning" which is a precise form of it. I'm not sure if we should clarify that or not.

Reply:
I'm not saying this phrasing is a problem, just thinking out loud

Reply:
I've always felt a little uncomfortable about this - I'm happy to drop the phrase. We should define a versioning strategy.

Reply:
I've replaced the phrase "semantic versioning" with "a form of semantic versioning". If there is a general consensus this is potentially misleading, replacement phrases are welcome.